frontend/src/types/data.ts

```diff
@@ -16,3 +16,9 @@ export type Token = {
   symbol: string;
   decimals: number;
 };
+
+export const ETH: Token = {
```

Comment:
I doubt this a good place for this, but I don't care too much right now. Just a nitpick...